Updated to work with Suave 1.1.3 and other package updates

emmanuelchevallier opened the pull request:
Also include an update for a future breaking with Suave 2.0

https://github.com/SuaveIO/suave/commit/538a445524a208a6e2c9fa77647fc2ecc934aaea

emmanuelchevallier committed:
Updated to work with Suave 1.1.3 and other package updates

Also include an update for a future breaking with Suave 2.0

https://github.com/SuaveIO/suave/commit/538a445524a208a6e2c9fa77647fc2ecc934aaea

tpetricek commented:
Nice, thanks!

Can you add `framework: >= net40` as the first line in `paket.dependencies`? That way, Paket will not be downloading all the .NET Core dependencies and `paket.lock` will be a lot smaller.

Also, I think there is no need to keep the old version in comments - if you can just update everything to the new version, that's ideal.

emmanuelchevallier committed:
Correction as recommended by @tpetricek

https://github.com/tpetricek/suave-fshome/pull/2

tpetricek commented on app.fsx:
```diff
@@ -45,7 +46,10 @@ let getFeedNews () = async {
+        //error FS0001: This expression was expected to have type string option
+        //yield sprintf "<p>%s</p>" (formatText 500 (stripHtml (defaultArg item.Description "")))
+        //Replaced by code found at https://github.com/tpetricek/suave-fshome/issues/1
+        yield sprintf "<p>%s</p>" (formatText 500 (stripHtml (defaultArg (Some item.Description) "")))
```
`defaultArg (Some item.Description) ""` can be replaced by just `item.Description`.

emmanuelchevallier committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into tpetricek/master

emmanuelchevallier committed:
Remove commented line and optimize packet managment

Remove commented lines even for future breaking in Suave 2.0 that will
be post in an issue, to keep a trace.

Adding framework: >= net40 in paket.dependencies as @tpetricek
recommanded that have reduced a lot the paket.lock and package needed to
be downloaded.

emmanuelchevallier commented:
Thanks to you Tomas for this project and all the work you do for F# and the community !

I did the modifications, removed commented lines and the package management is definitely better ;)

As a recently graduated medical doctor with a past in computer science I think F# should have an important place in healthcare. I just began to develop in F# (I come from R, PHP and a little bit of Webforms a long time ago...) it's hard but I hold on ! F# is awesome !

tpetricek commented:
Thank you - this is now perfect! It's great to see people with diverse backgrounds using F# and I'm glad some of my work helps that :-).
